pkgs/misc/cups/drivers/brother/mfc9332cdw/default.nix

```diff
@@ -0,0 +1,150 @@
+{ pkgsi686Linux
+, stdenv
+, fetchurl
+, dpkg
+, makeWrapper
+, coreutils
+, ghostscript
+, gnugrep
+, gnused
+, which
+, lib
+, gawk
+, a2ps
+, file
+}:
+
+let
+  model = "mfc9332cdw";
+  relpath_printer = "opt/brother/Printers/${model}";
+in rec {
+  mfc9332cdwlpr = pkgsi686Linux.stdenv.mkDerivation rec {
+    version = "1.1.3-0";
+    name = "${model}drv-${version}";
+    dlfid = "dlf101620";
+    relpath_lpd = "${relpath_printer}/lpd";
```

Comment:
move these into the above let expr, so it's more readable what other variables are available inside the `mkDerivation`.